README.md

```diff
@@ -54,9 +54,9 @@ O-IL8-15 Biological Probe, in Structural Genomics Consortium: [thesgc.org/biolog
 
 ---
 ## AlphaFold2
-AlphaFold2 is a deep learning system that predicts protein structures from amino acid sequences. We used the open-source distribution of AlphaFold2, [ColabFold](https://github.com/sokrypton/ColabFold) to predict the structure of the antibody. In particular, we used the [AlphaFold2_mmseqs2](https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb) notebook. This notebook differs from full AlphaFold2 and AlphaFold2 Colab in that it uses MMseqs2 (Many-against-Many sequence searching) in place of homology detection and MSA pairing.
+AlphaFold2 is a deep learning system that predicts protein structures from amino acid sequences. We used the open-source distribution of AlphaFold2, [ColabFold](https://github.com/sokrypton/ColabFold) to predict the structure of the antibody. In particular, we used the [AlphaFold2_mmseqs2](https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb) notebook. This notebook differs from full AlphaFold2 and AlphaFold2 Colab in that it uses MMseqs2 (Many-against-Many sequence searching) in place of homology detection and MSA pairing. MMseqs2 can predict complex structures with great accuracy, as compared to AlphaFold2.
```

Comment:
this addition seems a little inaccurate. Can you mention a source in the comments?

Reply:
Yes, I reviewed the references, and this sentence has a flaw. It should be about speed and complexity, rather than accuracy, and it should be MSA pairing rather than AlphaFold2.
https://www.nature.com/articles/s41592-022-01488-1
https://computationalbiomed.hms.harvard.edu/tools-technologies-old/alphafold-colabfold/

Reply:
Okay, cool! Why not also cite the source ?

Reply:
In the citations part, right?